api/src/test/java/io/opentelemetry/api/baggage/BaggageContextTest.java

```diff
@@ -24,7 +24,7 @@ void testGetCurrentBaggage_Default() {
   @Test
   void testGetCurrentBaggage_SetCorrContext() {
```

Comment:
this test method name is pretty out of date..still referring to it as "correlationContext"